exporter/tanzuobservabilityexporter/README.md

```diff
@@ -61,6 +61,10 @@ reduce memory
 consumption. [Details and defaults here](https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/memorylimiterprocessor/README.md)
 .
 
+Note: Enable memory limiter processor(The order of enabled processors matters. Please refer to the
```

Comment:
I don't understand what you're trying to convey here – maybe there's missing punctuation?

I was thinking:

**Note:** The order matters when enabling multiple processors in a pipeline (e.g. the memory limiter and batch processors in the example config below). Please refer to to the processors' [documentation](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor) for more information.

Reply:
fixed